packages/core/examples/webLoader/registerWebImageLoader.ts

```diff
@@ -70,11 +78,12 @@ function createImage(image, imageId) {
     height: rows,
     width: columns,
     color: true,
-    rgba: true,
+    // since the canvas return rgba we should tell the cornerstone that we have 4 components per pixel
```

Comment:
I don't get this comment - it implies that we should be setting rgba to true is how I read it.